share/smack/frontend.py

```diff
@@ -95,8 +95,6 @@ def default_clang_compile_command(args, lib=False):
         cmd += ['-DFLOAT_ENABLED']
     if args.pthread:
         cmd += ['-DSMACK_MAX_THREADS=' + str(args.max_threads)]
-    if args.bit_precise:
```

Comment:
How come we don't need this any more?

Reply:
Good catch. I grepped the wrong pattern. We did need it.